_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -159,10 +169,21 @@ proc cmake::handle_generator {option action args} {
             "*Ninja" {
                 ui_debug "Selecting the Ninja generator ($args)"
                 depends_build-append \
-                                port:ninja
+                                path:bin/ninja:ninja
+                depends_skip_archcheck-append \
+                                ninja
```

Comment:
The ninja Portfile already specifies `installs_libs no` so using `depends_skip_archcheck` for ninja here should not be necessary.